RealmSwift/LinkingObjects.swift

```diff
@@ -436,10 +436,3 @@ internal enum LinkingObjectsBridgingMetadata {
         }
     }
 }
-
-// MARK: Unavailable
```

Comment:
Because the migration assistance this provides is already provided by the more generic ones on `RealmCollection`, so it's redundant to add it here as well.